driver/src/test/java/org/neo4j/driver/util/DatabaseExtension.java

```diff
@@ -174,13 +160,15 @@ public void stopDb()
         runner.stopNeo4j();
     }
 
-    public int getMajorVersion()
+    public boolean isNeo4j43OrEarlier()
     {
         try ( Session session = driver().session() )
         {
             String protocolVersion = session.readTransaction( tx -> tx.run( "RETURN 1" ).consume().server().protocolVersion() );
```

Comment:
I would rather use 

```
CALL dbms.components() YIELD versions, edition 
RETURN 'Neo4j/' + versions[0] AS version, edition
```

bolt protocol depends on what server and client negotiated.